src/metabase/util/schema.clj

```diff
@@ -288,6 +290,15 @@
                                   (deferred-tru "Valid field semantic or relation type (keyword or string)"))
     (deferred-tru "value must be a valid field semantic or relation type (keyword or string).")))
 
+(def FieldRef
+  "Schema for a valid Field Referenced."
+  (s/cond-pre
+    mbql.s/FieldOrAggregationReference
+    (s/pred
+      (comp (complement (s/checker mbql.s/FieldOrAggregationReference))
+            mbql.normalize/normalize-tokens)
+      "Field or aggregation reference as it comes in to the API")))
```

Comment:
Why do we need this schema? Because the tokens are normalized on the BE, but not the FE?

Reply:
Ummm, I'm not sure. I just moved this over from `src/metabase/sync/analyze/query_results.clj`

Reply:
It says why in this comment 
https://github.com/metabase/metabase/blob/769bba1ade874c373eef23dc6162edd0f2f4bbcf/src/metabase/sync/analyze/query_results.clj#L28

In uses of `ParameterSourceOptions`, are the tokens normalized already, or not? I'm not sure you need both options as in `src/metabase/sync/analyze/query_results.clj`

Reply:
you're right. I revert the move and create a new on with with only the checking part for API usage.
https://github.com/metabase/metabase/pull/27343/commits/3e1be713bd80bd34fa83088f0682430ea401e35b